src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/IssueNumberRetriever.java

```diff
@@ -11,10 +11,14 @@
 @Service
 public class IssueNumberRetriever {
 
-    public static final String GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE = "select nextval('MAILING_ISSUE_NUMBER_SEQ') as issue";
+    private static final String GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE = "select nextval('MAILING_ISSUE_NUMBER_SEQ') as issue";
+    private static final String GET_CURRENT_FROM_ISSUE_NUMBER_SEQUENCE = "select last_value as issue from MAILING_ISSUE_NUMBER_SEQ";
```

Comment:
1. If we have NEXTVALUE, we should also have CURRENT_VALUE to have consistent naming